### ISSUE TITLE
Won't load on newer alpha 0.0.8 versions

### ISSUE THREAD

**Issue by mikejgray** [OpenVoiceOS/ovos-mplayer-plugin#1]
Won't load on newer alpha 0.0.8 versions

```sh
2023-10-21 23:25:05.992 - audio - ovos_plugin_manager.utils:find_plugins:115 - ERROR - Failed to load plugin entry point EntryPoint(name='ovos_mplayer', value='ovos_plugin_mplayer', group='mycroft.plugin.audioservice'): multiple exception types must be parenthesized (__init__.py, line 799)
```

**Comment by mikejgray** [comment]
A new error:

```
2023-12-25 20:09:29.429 - audio - ovos_plugin_manager.utils:find_plugins:118 - ERROR - Failed to load plugin entry point EntryPoint(name='ovos_mplayer', value='ovos_plugin_mplayer', group='mycroft.plugin.audioservice'): invalid syntax (__init__.py, line 799)
2023-12-25 20:09:29.434 - audio - ovos_audio.audio:load_services:95 - INFO - Loading audio service plugin: ovos_common_play
2023-12-25 20:09:29.437 - audio - ovos_plugin_common_play:create_ocp:30 - DEBUG - OCP config:
 {'active': True,
 'autoplay': True,
 'max_timeout': 10,
 'min_timeout': 3,
 'mode': 'auto',
 'playback_mode': 40,
 'preferred_audio_services': ['vlc', 'mplayer'],
 'type': 'ovos_common_play'}
```

**Comment by mikejgray** [comment]
@JarbasAl Might fit in the OCP sprint?

**Comment by JarbasAl** [comment]
the dependency of this package was for py2, this never worked in ovos lol, just a code backup

fixed by #2
